bin/git_hooks/pre-push

```diff
@@ -4,7 +4,7 @@ green="\033[0;32m"
 color_off="\033[m"
 
 printf "\n==================== Running flake8 ====================\n"
-flake8=`docker run --rm -v $(pwd)/src/backend:/app trailhead_django /usr/local/bin/flake8`
+flake8=`docker run --rm -v $(pwd)/src/backend:/app $(basename $(pwd))_django /usr/local/bin/flake8`
```

Comment:
This may work as well?
```suggestion
flake8=`docker-compose exec django flake8`
```

Maybe there was a reason it was done this way, though..

Reply:
Yeah we did that so that if you're currently running one Skeletor-based project and want to push something real quick on another Skeletor-based project, there won't be any weird conflicts between docker containers. If that makes sense.

Reply:
Ah okay neat!